src/pool/cli.c

```diff
@@ -621,6 +628,17 @@ pool_disconnect_cp(tse_task_t *task, void *data)
 
 	pl_map_disconnect(pool->dp_pool);
 
+	rc = dc_mgmt_pool_disconnect(pool);
+	if (rc != 0) {
+		/* Its not fatal if we dont notify the agent of the disconnect
```

Comment:
Its -> It's or It is
dont -> don't or do not.  I think you'll get a checkpatch failure because of this typo.

Reply:
Oddly checkpatch didn't catch either of these. Fixed.